src/extensions/styles/store/reducer.js

```
@@ -58,57 +60,69 @@
-									stylesObj,
-									isIframe,
-									isSiteEditor,
-									breakpoint
-								),
-							}),
-							{}
-						),
-					},
-				},
+			const breakpointStyles = BREAKPOINTS.reduce((acc, breakpoint) => ({
+				...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)